test/specs/components/action.spec.jsx

```diff
@@ -94,6 +87,9 @@ xdescribe('Action', () => {
 
         sandbox.stub(conversationService, 'postPostback');
         conversationService.postPostback.resolves();
+
+        sandbox.stub(appUtils, 'getIntegration');
```

Comment:
Yeah... `getIntegration` will be tested in the `appUtils` unit test. Although, there's not test for that yet...